### PR TITLE
person right side

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

File: src/app/persons/person/person.component.ts [hunk 1]
```diff
@@ -19,14 +23,17 @@ export class PersonComponent implements OnInit {
   personId: string;
   myFormElements: FormBase<any>[] = [];
   formData = FORM_DATA_PERSON;
-
+  subscriptions: Subscription[] = [];
```
Comment (adriandumavitagroup) [inline src/app/persons/person/person.component.ts]: why this?  subscriptions: Subscription[] = []; .... if it is not used?

File: src/app/persons/person/person.component.ts [hunk 2]
```diff
@@ -41,6 +48,7 @@ export class PersonComponent implements OnInit {
         setTimeout(() => {
           this.sendResourceService.setResource(this.person, SentResourceTypes.PERSON_DATA);
```
Comment (adriandumavitagroup) [inline src/app/persons/person/person.component.ts]: why is this necessary?

Comment (sandorbanvitagroup) [reply src/app/persons/person/person.component.ts]: Because in the addresses-list-component we need to have some info about the person.

Comment (sandorbanvitagroup) [reply src/app/persons/person/person.component.ts]: But this wasn't added in this PR.

Comment (adriandumavitagroup) [reply src/app/persons/person/person.component.ts]: cool

File: src/app/persons/person/person.component.ts (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/persons/person/person.component.ts]: Resolved.

File: src/app/cases/cases-list/cases-list.component.ts [hunk 3]
```diff
@@ -105,7 +105,14 @@ export class CasesListComponent implements OnInit, OnDestroy {
       width: CASE_EXPORT_CUSTOM_MODAL_WIDTH,
     });
   }
-
+  setQueryParamsInFilters(): Filter[] {
```
Comment (danielrus) [inline src/app/cases/cases-list/cases-list.component.ts]: we could move this into helper service, so we don't have to duplicate it

File: src/app/shared/collapsable-box/collapsable-box.component.html [hunk 4]
```diff
@@ -11,9 +11,17 @@
...
     </mat-panel-title>
-    <mat-panel-description *ngIf="actionText">
+    <mat-panel-description *ngIf="actionText && !showGroupLink">
       <button mat-button color="primary" (click)="doAction($event)">{{ actionText }}</button>
     </mat-panel-description>
   </mat-expansion-panel-header>
   <ng-content></ng-content>
+  <app-card-group-link *ngIf="showGroupLink">
+    <mat-panel-description *ngIf="actionText">
+      <!-- <button mat-button color="primary" (click)="navigate(actionLink)">{{ actionText }}</button> -->
```
Comment (danielrus) [inline src/app/shared/collapsable-box/collapsable-box.component.html]: needs cleanup

File: src/app/shared/card/card-group-link/card-group-link.component.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-card-group-link',
+  templateUrl: './card-group-link.component.html',
+  styleUrls: ['./card-group-link.component.scss'],
+})
+export class CardGroupLinkComponent {}
```
Comment (danielrus) [inline src/app/shared/card/card-group-link/card-group-link.component.ts]: i don't understand the purpose of this component;
it seems to be an empty wrapper, no functionality or style applied to content

File: src/app/cases/cases-list/cases-list.component.ts (hunk 3 above)
Comment (sandorbanvitagroup) [reply src/app/cases/cases-list/cases-list.component.ts]: Done.

File: src/app/shared/card/card-group-link/card-group-link.component.ts (hunk 5 above)
Comment (sandorbanvitagroup) [reply src/app/shared/card/card-group-link/card-group-link.component.ts]: I thought it will have some settings, but you're right. It wasn't necessary. Removed.

File: src/app/shared/collapsable-box/collapsable-box.component.html (hunk 4 above)
Comment (sandorbanvitagroup) [reply src/app/shared/collapsable-box/collapsable-box.component.html]: Done.

**Review by danielrus — DISMISSED** [review]

**Review by danielrus — DISMISSED** [review]

File: src/app/persons/person/person.component.html [hunk 6]
```diff
@@ -15,7 +15,40 @@ <h1>
         [formElements]="myFormElements"
         [resourceService]="personService"
       ></app-dynamic-form>
-      <div aside>sidebar</div>
+      <div aside>
+        <app-edge-panel
+          type="CASE"
```
Comment (adriandumavitagroup) [inline src/app/persons/person/person.component.html:20]: maybe constants here, at type?

Comment (sandorbanvitagroup) [reply src/app/persons/person/person.component.html:20]: No needed here.

**Review by danielrus — APPROVED** [review]
